python/django_bolt/auth/user_loader.py

```
@@ -103,18 +111,18 @@ def load_via_sync(user_id: str, auth_context: dict | None, is_async_context: boo
 
             def load_via_plain(user_id: str, auth_context: dict | None, is_async_context: bool) -> Any:
                 if is_async_context:
-                    return _get_executor().submit(get_user, user_id, auth_context or {}).result()
+                    return _run_orm_blocking(get_user, user_id, auth_context or {})
                 return get_user(user_id, auth_context or {})
 
             return load_via_plain
 
         def load_via_async(user_id: str, auth_context: dict | None, is_async_context: bool) -> Any:
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)